flujo/validation/fixers.py

```
@@ -0,0 +1,425 @@
+        pat = _re.compile(r"(\|\s*)" + _re.escape(bad) + r"\b")
+
+        def _sub(m: _re.Match[str], replacement: str = good) -> str:
+            nonlocal changed
+            changed += 1
+            return m.group(1) + replacement
+
+        new_text = pat.sub(_sub, new_text)
+    return new_text, changed
+
+
+def _vt3_apply(file_path: str, report: Any, assume_yes: bool) -> FixResult:
```

Ruff (0.12.2)

Unused function argument: `report`

(ARG001)